notebooks/uncertainty_aware_models/labs/uncertainty_quantification_with_sngp_bert.ipynb

```diff
@@ -44,14 +44,12 @@
    ]
```

Comment:
<p>Should add an instruction of how to create this kernel?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done!